package.json

```diff
@@ -33,13 +33,16 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.30.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
+    "fastify": "^4.0.1",
+    "fastify-overview": "^2.2.0",
```

Comment:
I did not update yet the plugin to fastify v4: I can do this bump in a separate PR